flink-streaming-java/src/main/java/org/apache/flink/streaming/api/functions/query/AbstractQueryableStateOperator.java

```diff
@@ -28,16 +28,17 @@
 /**
  * Internal operator handling queryable state instances (setup and update).
  *
+ * @param <V>  Value type
  * @param <S>  State type
  * @param <IN> Input type
  */
 @Internal
-abstract class AbstractQueryableStateOperator<S extends State, IN>
+abstract class AbstractQueryableStateOperator<V, S extends State<V>, IN>
```

Comment:
I think here the same thing I mentioned earlier holds, we don't really need the additional `V` parameter.